packages/sqs-batch/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "@middy/sqs-batch",
+  "version": "1.0.0-beta",
+  "description": "SQs batch error handler middleware for the middy framework",
```

Comment:
```suggestion
  "description": "SQS batch error handler middleware for the middy framework",
```

Reply:
Ugh. The shift key on my shitty MacBook butterfly keyboard is going out. I'll fix this evening.